doc/sources/array_api.rst

```diff
@@ -0,0 +1,123 @@
+    If ```config_context(target_offload=device)``` enabled for DPCTL or DPNP inputs and device is as a string
+    for the device primarily used to perform computations, returned output will have different sycl queue for
+    the device. `target_offload` provided with `config_context` will be used to perform computations.
+
+
+Example usage
+=============
+
+DPNP ndarrays
+-------------
+
+Here is an example code to demonstrate how to use `dpnp <https://github.com/IntelPython/dpnp>`__ arrays to
+run `RandomForestRegressor` on a GPU witout ```config_context(array_api_dispatch=True)```:
+
+.. literalinclude:: ../../examples/sklearnex/random_forest_regressor_dpnp.py
+	   :language: python
+
+
+.. note::
+    Functional support doesn't guarantee that after the model is trained, fitted attributes that are arrays
+    will also be from the same namespace as the training data.
+
+For example, if `dpnp's <https://github.com/IntelPython/dpnp>`__ namespace was used for training,
+then fitted attributes will be on the CPU and `numpy.ndarray` data format.
+
+DPCTL usm_ndarrays
+------------------
+Here is an example code to demonstrate how to use `dpctl <https://intelpython.github.io/dpctl/latest/index.html>`__
+arrays to run `RandomForestClassifier` on a GPU witout ```config_context(array_api_dispatch=True)```:
+
+.. literalinclude:: ../../examples/sklearnex/random_forest_classifier_dpctl.py
+	   :language: python
+
+As on previous example, if `dpctl <https://intelpython.github.io/dpctl/latest/index.html>`__ Array API namespace was
+used for training, then fitted attributes will be on the CPU and `numpy.ndarray` data format.
+
+Use of `array-api-strict`
+-------------------------
+
+Here is an example code to demonstrate how to use `array-api-strict <https://github.com/data-apis/array-api-strict>`__
+arrays to run `DBSCAN`.
+
+.. literalinclude:: ../../examples/sklearnex/dbscan_array_api.py
+	   :language: python
+
+
+Support for Array API-compatible inputs
+=======================================
+All patched estimators, metrics, tools and non-scikit-learn estimators functionally support Array API.
+Functionally all input and output data have the same data format type.
+
+Stock scikit-learn uses ```config_context(array_api_dispatch=True)``` for enabling Array API `support <https://scikit-learn.org/1.5/modules/array_api.html>`__.
+If `array_api_dispatch` enabled and array api is supported for the stock scikit-learn, then raw inputs are used for the fallback.
+
+.. note::
+    Stock scikit-learn doesn't support DPCTL usm_ndarray or DPNP ndarray inputs, host numpy copies
+    of the inputs data will be used for the fallback cases. This fallback condition will be changed
+    in the next versions, depending on the support of Array API standard by DPNP and DPCTL and
+    reimplementation of `sklearnex` estimators via Array API.
+
+
+Planned features
+================
+It is planned to implement DBSCAN, PCA and other estimators via Array API for next releases.
```

Comment:
I'm not getting the meaning of this one.

Previously it said:
> All patched estimators, metrics, tools and non-scikit-learn estimators functionally support Array API.

What's the difference?